Assets/Scripts/receiveSignal/receiveSignalLockedDoor.cs

```diff
@@ -6,22 +6,27 @@ public class receiveSignalLockedDoor : receiveSignal {
     private Animator animator;
     //Currently I am just activating/deactivating a bool that must be on in order to swtich scenes. There is probably
     //a better way to do this though.
+	private Collider2D collider;
+
     public switchScene doorToNextLevel;
 
     void Start() {
         animator = GetComponent<Animator>();
+		collider = GetComponent <BoxCollider2D> ();
     }
 
 
     public override void activate() {
         animator.SetBool("open", true);
+		collider.enabled = false;
         if (doorToNextLevel != null) {
```

Comment:
You should see if you can get rid of the doorToNextLevel boolean. We can maybe just haved the collider get in the way so you can't interact with the door.